tdrs-backend/Pipfile

```diff
@@ -4,47 +4,46 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-mkdocs = "*"
+awscli = "*"
+awscli-local = "*"
 factory-boy = "*"
 flake8 = "*"
 flake8-docstrings = "*"
 isort = "*"
+mkdocs = "*"
 nose-progressive = "*"
+pygraphviz = "*"
+pyparsing = "*"
 pytest = "*"
 pytest-cov = "*"
 pytest-django = "*"
 pytest-mock = "*"
-pyparsing = "*"
-pygraphviz = "*"
-awscli = "*"
-awscli-local = "*"
 
 [packages]
-boto3 = "~=1.17.79"
-cryptography = "~=3.4.7"
-django = "~=3.2"
-django-configurations = "~=2.2"
-django-cors-headers = "~=3.7.0"
-django-extensions = "~=3.1.0"
-django-filter = "~=2.4.0"
-django-model-utils = "~=4.1.1"
-django-storages = "~=1.11.1"
-django-unique-upload = "~=0.2.1"
-djangorestframework = "~=3.12.2"
-dj-database-url = "~=0.5.0"
-gunicorn = "~=20.0.4"
-ipdb = "~=0.13.9"
-ipython = "~=7.24.1"
-jwcrypto = "~=0.8"
-markdown = "~=3.3.3"
-psycopg2-binary = "~=2.8.6"
+boto3 = "==1.17.112"
```

Comment:
I alphabetized this for OCD reasons but the notable change here is that the version numbers were all updated to match what is currently in the Pipfile.lock and set to use a strict equality for the version number. This ensures that future dependabot PRs for Python dependencies won't update unrelated packages when building a new lock file.

Reply:
Thank you for this! Driving me up a wall